run.sh

```diff
@@ -1,4 +1,15 @@
-source env/bin/activate;
+function setup_err(){
+  echo "Need to run setup.sh"
+  exit 1
+}
+source env/bin/activate || setup_err
```

Comment:
Please add a comment indicating that this calls setup_err() if if we can't activate the venv.